src/api.ts

```diff
@@ -126,15 +126,22 @@ export class OpenSeaAPI {
    * Get which version of Wyvern exchange to use to create orders
    * Simply return null in case API doesn't give us a good response
    */
```

Comment:
I believe we should update this doc since we added new logic in `try/catch` block.

Reply:
Done